libs/client/dashboard/src/lib/pages/tags-management/tags-management.component.ts

```diff
@@ -19,20 +19,15 @@ export class TagsManagementComponent implements OnInit {
     ) {}
 
     ngOnInit(): void {
-        this.tagsService.fetchTags(TagKind.Fandom).subscribe();
+        this.tagsService.fetchTagsSortedByParent(TagKind.Fandom).subscribe();
     }
 
     createTag() {
         this.dialog.open(TagFormComponent);
     }
 
     addChild(parentId: string) {
-        const alertData: PopupModel = {
-            message: 'Currently disabled',
-            confirm: true,
-        };
-        const dialogRef = this.dialog.open(PopupComponent, { data: alertData });
-        // this.dialog.open(ChildTagFormComponent, { data: { parentId: parentId }});
+        this.dialog.open(ChildTagFormComponent, { data: { parentId: parentId }});
```

Comment:
This comes back to what I was talking about in Discord--I don't think we want to pigeonhole ourselves into "one parent, one child" here. I think we should just have an "add tag" form, and that form can have an optional field for "parent". That'll allow us arbitrarily-deep nesting (the whole point of my PR), and if we want to, we can add depth limitations on the backend (or even the frontend, depending on how much it knows about the whole tree).

Reply:
In the future, if we want to go to arbitrary fandom tag levels, then I think it's worth redesigning the UI in the future to accommodate our needs. In the meantime, having clear UI that just handles two levels seems like it would be better for now. We need to account for usability with the mods that will be using this.